core/toolbar.go

```diff
@@ -217,10 +217,10 @@ func ToolbarStyles(w Widget) {
 	wb.FinalStyler(func(s *styles.Style) {
 		if s.Direction == styles.Row {
 			s.Grow.Set(1, 0)
-			s.Padding.SetHorizontal(units.Dp(ConstantSpacing(16)))
+			s.Padding.SetHorizontal(units.Dp(16))
```

Comment:
why is this in here and what is the motivation?

Reply:
If this is to improve alignment in the table, which it seems like from your comment, then it should be styled specifically in the table, not in the generic toolbar styles.  For those using compact spacing, it is essential to have this constant spacing for the general toolbar case.

Reply:
With our new border radius based parent border radius encroachment fix (#660), we don't need constant spacing on the toolbar anymore.

Reply:
The table *does* now have specific styling, which is zero padding, but again, the toolbar does not need to have the constant spacing anymore.

Reply:
ok, I tested it on various use-cases and toolbars look fine..

Reply:
So did I.